src/service/world-api-adapter.js

```diff
@@ -0,0 +1,265 @@
+/**
+ * ##World API Adapter
+ *
+ * The World API Adapter allows you to create, access, and manipulate multiplayer worlds its users and runs
+ *
+ * All API calls take in an "options" object as the last parameter. The options can be used to extend/override the World API Service defaults.
+ *
+ *
+ */
+
+'use strict';
+
+var ConfigService = require('./configuration-service');
+var StorageFactory = require('../store/store-factory');
+// var qutil = require('../util/query-util');
+var TransportFactory = require('../transport/http-transport-factory');
+var _pick = require('../util/object-util')._pick;
+
+var apiEndpoint = 'multiplayer/game';
+
+module.exports = function (config) {
+    var store = new StorageFactory({ synchronous: true });
+
+    var defaults = {
+
+       token: store.get('epicenter.project.token') || '',
+
+       project: '',
+
+       account: '',
+
+       group: '',
+
+//        apiKey: '',
+
+//        domain: 'forio.com',
+
+        //Options to pass on to the underlying transport layer
+        transport: {},
+
+        success: $.noop,
+
+        error: $.noop
+    };
+
+    var serviceOptions = $.extend({}, defaults, config);
+    var urlConfig = new ConfigService(serviceOptions).get('server');
+
+    var transportOptions = $.extend(true, {}, serviceOptions.transport, {
+        url: urlConfig.getAPIPath(apiEndpoint)
+    });
+
+    if (serviceOptions.token) {
+        transportOptions.headers = {
+            'Authorization': 'Bearer ' + serviceOptions.token
+        };
+    }
+
+    var http = new TransportFactory(transportOptions);
+
+    var setFilterOrThrowError = function (options) {
```

Comment:
The only 'filter' allowed here is the id right? Maybe make more specific? Or are there other filters allowed?